dhall/src/Dhall/Pretty/Internal.hs

```diff
@@ -1164,14 +1164,16 @@ escapeText_ text = Text.concatMap adapt text
 
 prettyToString :: Pretty a => a -> String
 prettyToString =
-    Pretty.renderString . Pretty.layoutPretty options . Pretty.pretty
-  where
-   options = Pretty.LayoutOptions { Pretty.layoutPageWidth = Pretty.Unbounded }
+    Pretty.renderString . Pretty.layoutPretty layoutOpts . Pretty.pretty
```

Comment:
Uh, should we maybe also use `layoutSmart` consistently?!
```suggestion
    Pretty.renderString . Pretty.layoutSmart layoutOpts . Pretty.pretty
```

Reply:
Yeah, that's fine